src/composition.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 the Interpoli Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use alloc::{string::String, vec::Vec};
+use core::ops::Range;
+use kurbo::{PathEl, Shape as _};
+
+#[cfg(feature = "std")]
+use std::collections::HashMap;
+
+use crate::{animated, Brush, Repeater, Stroke, Transform, Value};
+
+/// Model of a Lottie file.
+#[derive(Clone, Default, Debug)]
+pub struct Composition {
+    /// Frames in which the animation is active.
+    pub frames: Range<f64>,
+    /// Frames per second.
+    pub frame_rate: f64,
```

Comment:
Improvement: The interval of frames should not be fixed in the composition, but encoded on the frames themselves.
Filed as #17